docs/README.simple-topo.loopback.md

```diff
@@ -0,0 +1,168 @@
+# Using a simple routing topology as below this demonstrates SONiC VPP Loopback
+
+  |-------------------------------------------------------------|
+  |                                                             |
+Host1 --------------------- Sonic-VPP-Router ---------------- Loopback0
+
```

Comment:
Loopback0 is another routing interface just like Ethernet0,1 . So it should be part of SOnic-VPP-Router in the diagram. In Place of Loopback0 Host2 should be place. 
The idea being Loopback0 is reachable from both the subnets.

Reply:
I accidentally added an older version of this file. Please find the appropriate version now.